docs/languages.md

```diff
@@ -0,0 +1,59 @@
+# Supported Languages
+Haste-server supports all 38 "common" languages according to [highlight.js](https://highlightjs.org/), the syntax library used for this project.
+Any extra languages originally put into this project have remained as well for a total of 47 languages.
+
+Below is a table of every supported language and it's shorthand if it has one. The shorthand can be used as a file extension.
+(For example haste.zneix.eu/ovanazeyad.rust could be re-written as haste.zneix.eu/ovanazeyad.rs).
+It is also perfectly valid to just use the name of the language (haste.zneix.eu/sadf.haskell)
```

Comment:
Please add `https://` to each of those example links, so they are shown as links.